### PR TITLE
Created Eth-Global-Guide.md and added structure

### PR DESCRIPTION
**Pull request by vanshika-srivastava**

### CODE REVIEW

File: Eth-Global-Guide.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- Create a simple Omnichain Contract
+- Create a simple Cross Chain Messaging Contract
+
+
+## FullStack tutorial 
+- Bitcoin Frontend (Omnichain)
+- CCM
+
+## Developer Resources
+- Installing Zetachain CLI
```
Comment (fadeev) [inline Eth-Global-Guide.md:27]: If you mean installing [`zetacored`](https://github.com/zeta-chain/node), then I think it's a bit out of scope for regular developers. You don't need a ZetaChain node running locally to develop omnichain contracts.

File: Eth-Global-Guide.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- Sending and Depositing Tokens
+- Smart Contract Template
+
+## Smart Contracts
+- Create a simple Omnichain Contract
+- Create a simple Cross Chain Messaging Contract
+
+
+## FullStack tutorial 
+- Bitcoin Frontend (Omnichain)
```
Comment (fadeev) [inline Eth-Global-Guide.md:23]: The [current Bitcoin  `index.html` tutorial](https://www.zetachain.com/docs/developers/omnichain/tutorials/bitcoin-frontend/) might be a bit too simplistic. Maybe in this set of guides we skip the frontend for now and focus on smart contract development.

File: Eth-Global-Guide.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Build with Zeta
+- Start building dapps ( Refer to the first half -  https://www.zetachain.com/docs/developers/overview/) 
+- Using Zetachain Smart Contracts (conceptual explanation)
+> - Omnichain Contracts
+> - Cross Chain Messaging (explanation 
+- Sending and Depositing Tokens
+- Smart Contract Template
+
+## Smart Contracts
```
Comment (fadeev) [inline Eth-Global-Guide.md:17]: CCM and omnichain are quite different in terms of stack, so I'd do:

* Introduction
* Build with ZetaChain (things common to both CCM and omnichain)
  * Start building dapps
  * Sending and depositing tokens
  * Smrt contract template
* Omnichain contracts
* Cross-chain messaging
* Developer resources

File: Eth-Global-Guide.md (hunk 2 above)
Comment (vanshika-srivastava) [reply Eth-Global-Guide.md:23]: Makes sense ! maybe we can add this as a hyperlink under Omnichain contract and redirect towards the doc directly !

File: Eth-Global-Guide.md (hunk 3 above)
Comment (fadeev) [reply Eth-Global-Guide.md]: btw, the introduction section was fine:

## Introduction

- What is ZetaChain?
- Why build dApps on Zeta ? (Key features) 
- What is ZETA token ?

**Pull request merged**